client/tailwind.config.js

```diff
@@ -5,6 +5,29 @@ module.exports = {
     extend: {
       colors: {
         "sc-blue": "#0077c0",
+        "sc-dark-blue": "#00629e",
+      },
+      keyframes: {
```

Comment:
the custom animations and keyframes are great, but consider moving these animations into a separate CSS file for better organization and easier maintenance since they are specific to NotFound page.

Reply:
Oh so this is what you meant yesterday. I understand. Would creating a "NotFound.css" in the NotFound folder be best practice here?

Reply:
if is this super specific to it then I would